doc/rtd/explanation/hardening.rst

```diff
@@ -0,0 +1,80 @@
+Security Hardening
+******************
+
+Cloud-init's use case is automating cloud instance initialization, with support
+across distributions and platforms. There is a myriad of ways to harden this
+space.
+
+Follow the security hardening guidelines provided by the OSes and cloud
+platforms that your cloud-init configuration is targeting.
+
+Many cloud platforms provide SSH public keys in metadata which setup the
+default user with the appropriate configured means of access using
+SSH public/private key pairs.
+
+
+Updated packages
+================
+
+To ensure the available security fixes are applied to you VMs images upon
+launch, it is recommended to update the packages
```

Comment:
Let's make this a link in RST instead of spelling out the URL inline below 
```suggestion
launch, it is recommended by `Ubuntu security team guidelines` to update the packages
```